CHANGELOG.md

```diff
@@ -20,6 +20,29 @@ Code v99.99.999
 
 -->
 
+## [4.8.0](https://github.com/coder/code-server/releases/tag/v4.8.0) - 2022-10-24
+
+Code v1.72.1
+
+### Added
+
+- Support for the Ports panel which leverages code-server's built-in proxy. It
+  also uses `VSCODE_PROXY_URI` where `{{port}}` is replace when forwarding a port.
+  Example: `VSCODE_PROXY_URI=https://{{port}}.kyle.dev` would forward an
+  application running on localhost:3000 to https://3000.kyle.dev
+- support for `--disable-workspace-trust` CLI flag
+- support for `--goto` flag to open file @ line:column
+
+### Changed
+
+- Updated Code to 1.72.1
+
+### Fixed
+
+- Fixed bug in code-server (run with Docker) where PATH was always overwritten
```

Comment:
If I am thinking of the right thing this still happens in our default Debian-based image but now we should have Ubuntu images where this is not a problem.  So maybe we should move to features and say something like:

- Add Ubuntu-based images for Docker releases. If you run into issues with PATH being overwritten in Docker please try the Ubuntu image as this is a problem in the Debian base image.

Reply:
Good call, i'll update